src/main/resources/logback.xml

```diff
@@ -36,7 +36,7 @@
     <logger name="org.hibernate.SQL" level="ALL" />
     <logger name="org.hibernate.type.descriptor.sql" level="TRACE" />
     -->
-    <logger name="dwbh.api" level="INFO" />
+    <logger name="patio" level="DEBUG" />
```

Comment:
This enables SQL logs to appear, and before we should uncomment `logback.xml` to achieve the same behaviour. Do we want this?